dev-qt/qtcore/qtcore-5.6.9999.ebuild

```diff
@@ -25,6 +25,7 @@ DEPEND="
 RDEPEND="${DEPEND}"
 
 QT5_TARGET_SUBDIRS=(
+	doc
```

Comment:
No I wasn't waiting for that. I'm just very busy lately and haven't had the time to look into this. Sorry.